src/Parse.purs

```diff
@@ -200,8 +202,14 @@ branches expr' branch_ =
 varDefs :: SParser (Raw Expr) -> SParser (Raw VarDefs)
 varDefs expr' = keyword str.let_ *> sepBy1_try branch token.semi
    where
-   branch :: SParser (Raw VarDef)
-   branch = VarDef <$> (pattern <* equals) <*> expr'
+      branch :: SParser (Raw VarDef)
+      branch = do
+         p <- trace "before pattern" $ \_ -> pattern
```

Comment:
Eventually we'll want to remove this so probably worth putting a comment in like
"TODO: remove before merge"